lib/logstash/plugin_mixins/jdbc.rb

```diff
@@ -175,17 +179,20 @@ def execute_statement(statement, parameters)
     begin
       parameters = symbolized_params(parameters)
       query = @database[statement, parameters]
-      sql_last_start = Time.now.utc
+      sql_last_value = @use_column_value ? @sql_last_value : Time.now.utc
+      @warning_sent = false
```

Comment:
since we may introduce additional warnings, maybe this should be named something specific to the column retrieval?